src/components/Atoms/Select/index.stories.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react'
+import { ComponentStory, ComponentMeta } from '@storybook/react'
+import Select from '.'
+import { Divider, Grid, Typography } from '@mui/material'
+
+export default {
+  title: 'Atoms/Select',
+  component: Select,
+  argTypes: {},
+} as ComponentMeta<typeof Select>
+
+const Template: ComponentStory<typeof Select> = (args) => (
+  <div style={{ width: '250px' }}>
+    <Select {...args} />
+  </div>
+)
+
+function MItem() {
+  return (
+    <Grid container sx={{ width: '150px' }}>
+      <Grid item container justifyContent={'space-between'}>
+        <Grid item>
+          <Typography>Sohail</Typography>
+        </Grid>
+        <Divider
+          orientation="vertical"
+          flexItem
+          sx={{ border: '1px solid grey' }}
+        />
+        <Grid item>
+          <Typography>Softility</Typography>
+        </Grid>
+      </Grid>
+    </Grid>
+  )
+}
+export const PrimarySelect = Template.bind({})
+PrimarySelect.args = {
+  value: '1',
+  selectVariant: 'outlined',
+  formSize: 'medium',
+  items: [{ id: '1', content: <MItem /> }],
```

Comment:
Have these feature in component it self.
We need to get minimum data as input. And need to enable all variants using props.
You can have props as ("enablePreText", "enablePostText") by default have these as false. if enable show that variation like (preText divider Text)
`[
  {
    "preText":"Sohil",
    "text" :"Softility",
    "postText":"Dev",
    "val":"1"
  },
  {
    "preText":"Sai Kiran",
    "text" :"Softility",
    "postText":"Dev",
    "val":"2"
  }
]`

Reply:
modified